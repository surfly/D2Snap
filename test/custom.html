<div>
    <h1>Outer</h1>
    <custom-element>
        <h2>Inner</h2>
    </custom-element>
</div>