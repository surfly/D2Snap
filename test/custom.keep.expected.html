# Outer
<custom-element>
  ## Inner
</custom-element>